wormhole-connect/vite.config.ts

```diff
@@ -89,7 +89,12 @@ const plugins = [
 ];
 
 const optimizeDeps = {
-  include: ['@emotion/styled'],
+  include: [
```

Comment:
Forgot to mention, I"m adding two more libs to optimizeDeps to prevent the below intermittent error from MUI in local dev:
```
error: styled_default is not a function
```